<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Библиотека: регситрация читателя</string>
  </property>
  <widget class="QLabel" name="lb_upper">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Все поля обязательны для ввода.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите ФИО (допускаются буквы любого алфавита и пробелы):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_birhday">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите дату рождения в формате дд.мм.гггг</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_bday">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_address">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Введите адрес:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_address">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Ввод</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>650</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_contact">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите контактную информацию (номер телефона, email):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_contact">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_name">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_bday">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_success">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_address">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_wrong_contact">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
